jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TimDepositController.java

```diff
@@ -211,7 +216,7 @@ public synchronized ResponseEntity<String> depositTim(String jsonString, Request
         String obfuscatedTimData = TimTransmogrifier.obfuscateRsuPassword(odeTimData.toJson());
         stringMsgProducer.send(jsonTopics.getTimBroadcast(), null, obfuscatedTimData);
 
-        // Now that the message gas been published to OdeBradcastTim topic, it should be
+        // Now that the message gas been published to OdeBroadcastTim topic, it should be
```

Comment:
(non-blocking) "message gas" should probably be changed to "message has"

Reply:
updated https://github.com/CDOT-CV/jpo-ode/pull/113/commits/e3f1e3f7bcd915ff7b80a09cd7a425d0fe00ac42